beacon/validator/src/main/java/tech/pegasys/teku/validator/coordinator/publisher/MilestoneBasedBlockPublisher.java

```diff
@@ -39,7 +40,8 @@ public MilestoneBasedBlockPublisher(
       final BlockFactory blockFactory,
       final BlockImportChannel blockImportChannel,
       final BlockGossipChannel blockGossipChannel,
-      final BlockAndBlobsSidecarGossipChannel blockAndBlobsSidecarGossipChannel,
+      final BlobSidecarPool blobSidecarPool,
+      final BlobSidecarGossipChannel blobSidecarGossipChannel,
```

Comment:
I think we should make it all the way `Optional` as there is no NOOP version

Reply:
I created NOOP version, think it's cleaner than optional